dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/services/impl/JPAPersistenceManagerImpl.java

```diff
@@ -2683,18 +2688,18 @@ public int getJobInstanceTableVersion() throws Exception {
         return getJobInstanceTableVersion(getPsu());
     }
 
-    @FFDCIgnore(javax.persistence.PersistenceException.class)
+    @FFDCIgnore(PersistenceException.class)
     private int getJobInstanceTableVersion(PersistenceServiceUnit psu) throws Exception {
         if (instanceVersion != null)
             return instanceVersion;
 
         final EntityManager em = psu.createEntityManager();
 
+        // Check for the presence of v2 table support (presence of UPDATETIME column)
         try {
             Integer exec = new TranRequest<Integer>(em) {
                 @Override
                 public Integer call() throws Exception {
-                    // Verify that UPDATETIME column exists by running a query against it.
                     String queryString = "SELECT COUNT(x.lastUpdatedTime) FROM JobInstanceEntityV2 x";
                     TypedQuery<Long> query = em.createQuery(queryString, Long.class);
                     query.getSingleResult();
```

Comment:
Let's break this up into two separate try-catch blocks... one to see if we can make it to V2.. the next to V3.